src/clr-angular/forms/styles/_input.clarity.scss

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2016-2018 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+.clr-control-container {
+    .clr-input-wrapper {
+        white-space: nowrap;
+        height: calc(#{$clr-forms-baseline} * 4 + 1px); // This is strange but makes the vertical rhythm work properly
```

Comment:
I saw this issue in the demo. So weird. Not sure whats going on here.

Just a note that using display flex solves this issue. But we might have to test the whole thing again. We can then remove this hack.

Reply:
Looks like it fixes things, but yes need to do validation.